model_tests.rb

```diff
@@ -2147,6 +2147,18 @@ def test_zone_hvac_cooling_panel_osm
     result = sim_test('zone_hvac_cooling_panel.osm')
   end
 
+  def test_zone_hvac_evaporative_cooler_rb
+    result = sim_test('zone_hvac_evaporative_cooler.rb')
+  end
+
+  # def test_zone_hvac_evaporative_cooler_py
+  # result = sim_test('zone_hvac_evaporative_cooler.py')
+  # end
```

Comment:
TODO: once happy with he ruby test, the python one needs to be added now in this PR